tests/unit/test_browse.py

```diff
@@ -298,39 +310,46 @@ def test_create_browse_imagery_with_mocks(
         )
 
     def test_convert_singleband_to_raster_without_colortable(self):
-        ds = MagicMock(DataArray)
-        ds.__getitem__.return_value = self.data
+        """Tests convert_gray_1band_to_raster."""
+
+        return_data = np.copy(self.floatdata)
```

Comment:
See if there's an easy way to remove the float data and just convert from the ints